lighthouse-core/report/html/renderer/util.js

```diff
@@ -35,9 +35,11 @@ class Util {
    * @return {string}
    */
   static formatDisplayValue(displayValue) {
-    if (typeof displayValue === 'undefined') return '';
     if (typeof displayValue === 'string') return displayValue;
+    if (!displayValue) return '';
 
+    // Don't mutate the value
+    displayValue = [...displayValue];
```

Comment:
oh heh, I guess we might want to use them more than once huh :) thanks for the cleanup!

wanna just remove the mutations? `const template = displayValue[0]; ... of displayValue.slice(1)`

Reply:
add a `util-test.js` test for mutation?